app/src/Molecule.re

```diff
@@ -0,0 +1,39 @@
+open Atom;
+open Particle;
+
+module Styles = {
```

Comment:
Shouldn't this be inside of the `ProjectCard` module?

Reply:
You're right, especially so when there's going to be more molecules.